tools/NodeConfigCompare/configuration/cloud_formation_template.yaml

```diff
@@ -57,6 +57,11 @@ Parameters:
     Description: SNS email endpoint to receive step function state change alerts
     Type: String
     Default: "N/A"
+  Branch:
```

Comment:
1. Can you change the variable name to be more descriptive, something like `GithubBranch`. 
2. Any reason this needs to be defined at the top-level like this or could we just define it in the resource `CopyRedshiftConfigTestingCode` for now. Remember that customers will have edit access to this cloud-formation, so to use a specific branch they can make a change there as well. I think it's also a niché usecase which will not be used by a lot of customers, so I would say less complicated this is, the better.

Reply:
I felt this would remain the only place where customers give input to the CFT. So decided to add it here. But I agree this is a niche usecase . Ill remove the Branch parameter. Ill modify `CopyRedshiftConfigTestingCode` and add the branch here. Customers should edit this if they want branches other than main.

Reply:
Done. Tested and successfully created the stack

Test stack name for reference: T29-NodeConfigRevamp